src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/jobs/DeactivateLicencesService.kt

```diff
@@ -71,6 +74,13 @@ class DeactivateLicencesService(
           eventDescription = "${reason ?: "Licence deactivated automatically as it passed release date"} for ${licence.forename} ${licence.surname}",
         ),
       )
+
+      domainEventsService.recordDomainEvent(
+        LicenceDomainEventType.LICENCE_INACTIVATED,
```

Comment:
if moving the event type on to kind, it would be worth referring to it here: `licence.kind.inactivatedDomainEventType` as you're working over a list of generic licences

(It's unlikely that this would ever be called with variations but would make sense to avoid the hard coding)